static/js/i18n.ts

```diff
@@ -28,17 +30,20 @@ export const $t = intl.formatMessage;
 export const default_html_elements = Object.fromEntries(
     ["b", "code", "em", "i", "kbd", "p", "strong"].map((tag) => [
         tag,
-        (content_html) => `<${tag}>${content_html}</${tag}>`,
+        (content_html: string) => `<${tag}>${content_html}</${tag}>`,
     ]),
 );
 
-export function $t_html(descriptor, values) {
+export function $t_html(
+    descriptor: MessageDescriptor,
+    values?: Record<string, PrimitiveType | FormatXMLElementFn<string, string>>,
+): string {
     return intl.formatMessage(descriptor, {
         ...default_html_elements,
         ...Object.fromEntries(
             Object.entries(values ?? {}).map(([key, value]) => [
                 key,
-                typeof value === "function" ? value : _.escape(value),
+                typeof value === "function" ? value : _.escape((value || "").toString()),
```

Comment:
Use `??` rather than `||` so we don’t turn zeros into empty strings.

Or better, `_.escape(value?.toString())` seems to work.

Reply:
Changed to `_.escape(value?.toString())`.